### PR TITLE
add playlist feature

### PR DESCRIPTION
**Pull request by archit7448**
### What I do:

- Added playlist feature all 
- Add loaders and alert
- create playlist modal  
### Files to check JSX

### CODE REVIEW

File: src/utility/notification/utility-toast.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    autoClose: 1000,
+    hideProgressBar: false,
+    closeOnClick: true,
+    pauseOnHover: true,
+    draggable: true,
+    progress: undefined,
+    className: "toast-container",
+  });
+
+export { notifyMessage, notifySuccess, notifyError, notifyInfo };
```
Comment (itsmebipul00) [inline src/utility/notification/utility-toast.js:48]: appreciation: Nicely done with toast

**Review by itsmebipul00 — APPROVED** [review]
Good to merge

File: src/Components/VideoCard/videoCard.jsx [hunk 2]
```diff
@@ -1,22 +1,54 @@
...
 import "./videoCard.css";
 import { useNavigate } from "react-router-dom";
-export const VideoCard = () => {
-  const { filterData } = useData();
+import { useState } from "react";
+export const VideoCard = ({ prop }) => {
+  const { data } = prop;
+  const [show, setShow] = useState(false);
+  const { dispatch } = useData();
+  const token = localStorage.getItem("token");
```
Comment (iRohitGaur) [inline src/Components/VideoCard/videoCard.jsx:11]: suggestion: get the token in context once and maintain from there instead of getting it on every page

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Good to merge 👍🏻

**Pull request merged**